static/raw/modules.drawio

```diff
@@ -0,0 +1 @@
05Jw0d9aWM3y+wCXMSVgzxb/WOAQQbHPS9SZEImSr+FV+rR8jt4ZCZKuYSyV15XCFI/l43OHZBYIhbLgFiIjE8Gs/FUdLeBO8BiQmAam7c0HbEXFOVQ5GR5TT0xbl7LGu+eJIjwZzpKsMIQ2OtMPnavY70gYYQJOpAaVasyx70q+D+tLVG/4sGpNB9/6zH5WafHU+4dTw2U/rppIGPLDYsHpGuAobaLHDuV+Mt3aoveITE6HJYTk+Px6PB+l5HTgeBKVflHOSukNKPxg8T9tdPs3chcQlVgrXn8bIckuuc1hkaPBAPmjktAcM3KFyUU6XK9Hl4un34jxXtto7vTjT1aX7LgnS382shApRXcaUNkA5snXOlh8irfEUhViCZPF2FZFgm5jT0kNeWGpJVvmDwbbA/EE0gIuMZMOAOPVD4oi+a4fP2ufwzZDOxdbbo08o9/AoIx+A2QyAgRw/W8QWr6Hp+BmmMsagy/FzZDudQcsPQ2aazB4fMcujRe6T3gwDMLiJqAFdwHcfOXhvQDJFIDmeq45XKbFkVaXDo0ml+YZamvunkNwrZt21rdolZWxdiWinSHUvtlEcVssiO6y3wSHKtpv6zpad2JYpoHMK1NFU1ZpxnWtO+MYWDXAW0SZDpKFBUiLvQpghCwQmAe5QxFb1z3lhRBdX4iWP7XDoltrxx8Lw2JXmVUP2IHnRfqCNbvyOvQDfA1wqWt+0bYFzDGgVY6IhqHVAmi+uFBZhmjKeta3+dQ0JzL/gVEAxSxDBO0V9JMBGwCMzAGSXCophQMCC/pbRBuv8C//78t9W8N9SOZm64qvujkylN6Ul25rTkrvqP1ncpqsBguDcsWEHzioWUophESOA37DJV5IrPAxFBonYcGf+PURs0DAaW2LExgCGvLzF4qr6JOvZPdku+BhG6+Zirw4S8nHfpbnK53oPgSWTkhiQ+WpxjWd1sWmLq8J/p/YnCjwC3yWj8y3pVE1qfr0holwmotgF+t4G7OtrzTFhM3cXxLAsYX7Dwn3ykxiZNwxkt+T+Zdm4EmHbm9UJeEvIlTJH/vYS1cTrJHDFjEvzUHxf9WAYFVnss35HSRzxZ5Zx6petitpDrmlj1Tqi9+I190Dl1726TFXcoW8uoNgLioNXJV5oqnmgySMC3IyqeDfZHJIsJ34guocuhroMjsvg+CA01TI4+JQNyRZNQ5pVg/emxO79iOb1XUbJZZR8GJpqGSX+cgkDPOMEnrbwJseB3Zz4WAu49p8isuPUZYxcxsgHoamWMUJ2JPKT13JNY8Z4Ew26rGBbFjTfgCw8rxMCiiDNuk2wN0ex3NHgYWH5XJkM5khRIs4+ckLPG+d48Mfoty8sCNAoqtJYGmXsmBhcsnvfeQ1xDFWM2lEEMQxD1w60nryAQcICpkbWv/fVyWVsgmHUPBysd8xxIq9dt2xv7CpmcBx7PJ2C+p82WskTOseIR1BLSFALSRN0yMlq/KNxY+FQcS0421fESmSZKVxui01zv+TktyhFpHBRJSX5DWpBya+qZGXV7pa43DXVup8XiGScAcLHQ3jh5xEJKct8IJUViAo3yAuOllWdi0q83LTaYzpO9WDr86furrLA9LFz05/N0XQ2mjidEeru6emDQmq8+98bD/rEBOcQb34lJB902K0Di5FuCfA9DPLzJXAkyHP/e9POmJcGSO+DBC4q5D2hCxA8HHK8w0XXXHTNO9E1D/5XkmqeI+nElhMNsXtjz7d/RBHX6vkPLv6K0s9L7O0hvUrXFkiM7weEN8cTNKSZtZIkDnX3dzvQ/ZcYALtyY5m7C0QHFkwVqw5sRe6atn0NvMuubSddHwXA/X3HyoOBUOd5VR6MsB2gcp2UPlzJOZGIXHAZlTSh72RLTI5LiJL2pxgoIeqtPOZHSI4trxEGUx2L7in5rTNJanJE1KwdnWfCYRjtP6A/SVFRlxk9nEe6E6x2r7GLhIf2vknJgz3p3vwK2OKsgi7O2hT9+NZliqgIeKzzogJEeyWZYmmmtAlQn8Z9v9TvPNFF93CjYZpZSvbGhM2JY10jbxcvtI5K7YzZC/t6T6vDKNq2LJnxiAzFLhXdNAkaNp96uYtkYwrHO9/IB4YLdqVgualtO2mecpdV2WjunoOu21Zi6C5ch8yzitbocrk0g0BpWRw8RelOxIWrriMn/1eWxUlsxol5uAHRLfOqiEOXyOOQVoYIzI40CXa0K0BAxavwAwLuQyIm76Yni+Y0RS0cEBZJngFZhsnOsf2Q/XuOEIqbWwSTRIp6852LhDpAQnXJoW7JNYSEcibtTWWqlR9NIWUp9+dzdC33k92cTccZvkcGTsI02G4Q2cwOwDGUgbHheJaFyO2CqTF1bNtt4sXYr3aaaNrp3Db3Xuh0sCVot0DDVghK7nJniGUqD4Y23Kpn61Okhzrq1QLV0qPqmGFDfZC0Cn4ToEtLdirJvoc/dZ/ssXPRavscpWFe2rWWVec7ZZUGuk7eOVkDlEXVFoo09Q/n9/krrLcTfwO1N1HPmt830KTr9KB9Psi5ONhpOJ0fUl9g7JfRE+SRiyxP8dEJuAl5MI9y7Y19d+BCRVf/q04UwQgjO5LAC8SOhNjvZH0YlVyVFf4DogrvUdANLLZzQ4nsGhYqujJM64pcmDc2MDg19HYtrzi0Gr2N7B2fGwjXv/CZdmSxREnEvMlDfOQPeOTbKSndPHOJb0hLtvs9A43PLvEFCaDjnV7nC/bO0erfguhdnQc8NdvhCINvZtvM35goYqYm8DS5g45qx5RLAscH2eC2Ch1UgJKD8B1b3JqGqSt3Q97i9ld6RvuTOcb7ph4kYZVpho8wK+FmQQYy/jlhQsgQp5Wca3sc1xXXlSkSPwy+wErY+k7XIY0TVfrYUFuf9Y4VHbPvWe/p1MGtLOXbN0dsdlbgSSg/O3KbM7Ul8OZsTrs9NM65K31rtABHVtcWcOXxcsT+ZuhrnmJ6Vtd+wur0IQ0hvuP/</diagram></mxfile>
```

Comment:
Tip: We shall move raw assets out of `static` folder (into `assets/raw`) since currently they will be complied to production deployments